### ISSUE TITLE
the current documentation is confusing

### ISSUE THREAD

**Issue by alexeinz** [respeaker/Alexa#8]
current documentation doesnt seem related to the respeaker set up
if you do download the ipk and run it, it already has its own client and security id in the executable
how would you pass it the correct information, and where the creds file (alexa.json) should be located and what would be the correct syntax .

**Comment by aleifuu** [comment]
It's a bit confusing I think I must admit

on ReSpeaker itself, when you signed on web_auth :3000, it doesn't copy your cred(s) to the executable ... I think it's using one of ReSpeaker's team's credential for it 

maybe dev/ can clarify ?

**Comment by xiongyihui** [comment]
Yes, on ReSpeaker,  the alexa ipk already contains its credential, so it doesn't  require to setup a alexa developer account.

**Comment by alexeinz** [comment]
Thats is not correct! It will not connect since you cant authenticate with account thats not yours.
if you use the ipk, you need to open up alexa.sh and edit the fields or use alexa.json (of unknown format) in /etc/config ...
Please correct the explanation.

**Comment by xiongyihui** [comment]
What is `alexa.sh` ? @alexeinz

`/etc/config/alexa.json` is a json format file contains `refresh_token`.

**Comment by alexeinz** [comment]
i meant the alexa executable at /usr/sbin 
it didnt work for me to authenticate to alexa web service , it required opening it in vi and adding the correct client and secret 
this should be clarified

**Comment by xiongyihui** [comment]
What error do you get when you use the default client id and secret id? Generally, we don't have to modify `/usr/bin/alexa`

**Comment by xiongyihui** [comment]
There are [amazon alexa developer account](https://developer.amazon.com/alexa-voice-service) and [ amazon user account](http://alexa.amazon.com/) for this alexa application. The ipk is built with respeaker's amazon alexa developer account info, So we use the alexa application by just login in with our amazon user account.

**Comment by aleifuu** [comment]
@alexeinz , I can use the executables `alexa` provided by the ipk with no problem

My only issue is how do you use your own credential with the executable. I guess no other way but modify it

Now, I just need to enter my amazon's alexa client id and secret id into it, don't I ?

**Comment by ishotjr** [comment]
@alexeinz I don't know if the docs/binaries/etc. have changed since you opened this issue, but I finally received my ReSpeaker, many, many months after everyone else - despite being an "Ultra Crazy Early Bird" backer (which as far as I can tell just means you have your early support thrown back in your face by not receiving the free case that everyone else did!) - and I have found the whole experience pretty confusing, with various documentation-ish looking odds and ends all over the place, much of which ends up not working as expected/intuitively.  I wasted an inordinate amount of time getting Alexa to work due to prior experience with Alexa on Raspberry Pis, where you're required to set up and authorize a device etc.; it turns out this is not required for the ReSpeaker, since Seeed have baked it in, *but* this only works if you use the `192.168.100.1` address on the ReSpeaker's own network - not the address on your local network, which won't have been whitelisted in their account.  Because I was familiar with that error from past experience, I went through the process of setting up a new device etc. - which turned out to not be necessary - I'm now using it with Seeed's developer account and my Amazon account no problem (though it is quite laggy and somehow Dots all over the house can hear me while I try to test the ReSpeaker that can't seem to from more than 8-10 inches away)... 😩
